src/zcash/JoinSplit.cpp

```diff
@@ -198,7 +198,8 @@ class JoinSplitCircuit : public JoinSplit<NumInputs, NumOutputs> {
         uint64_t vpub_old,
         uint64_t vpub_new,
         const uint256& rt,
-        bool computeProof
+        bool computeProof,
+        uint256 *out_ephemeralSecretKey // Payment disclosure
```

Comment:
The name ``ephemeralSecretKey`` isn't used anywhere else, and could be confused with the symmetric encryption key. I suggest changing this to ``out_esk``. (The "s" doesn't officially stand for anything.)

Reply:
Ok